src/views/HomePage/index.js

```diff
@@ -1,3 +1,88 @@
-import HomePage from './HomePage';
```

Comment:
Great job @Lundii!

I can see that you brought in the content of Homepage.js into this file. I am confused, and I would like to know why you made the changes.

Reply:
It shouldn't be there at all. I will remove it now. Thanks for the observations.

Reply:
> Great job @Lundii!
> 
> I can see that you brought in the content of Homepage.js into this file. I am confused, and I would like to know why you made the changes.

Done